psamm/datasource/sbml.py

```diff
@@ -962,5 +983,11 @@ def write_model(self, file, model):
 
         self._add_gene_list(model_tag, gene_ids)
 
+        for p in params:
+            param_tag = ET.SubElement(params_list, self._sbml_tag('parameter'))
+            param_tag.set(self._sbml_tag('id'), 'P_'+self._transform_param(p))
+            param_tag.set(self._sbml_tag('value'), p)
+            param_tag.set(self._sbml_tag('constant'), 'true')
+
```

Comment:
Maybe put the code for generating parameter list in a separate commit? And then just keep the validator fixes to the existing code in this commit?